.ibm/pipelines/openshift-tests.sh

```diff
@@ -8,8 +8,8 @@ DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
 cleanup() {
   echo "Cleaning up before exiting"
-  helm uninstall ${RELEASE_NAME} -n ${NAME_SPACE}
```

Comment:
@subhashkhileri why did you remove it?

Reply:
The plan was to keep the instance running until another PR check is triggered, allowing us to inspect the cluster in case of failure. I've noticed that we've commented out `oc delete namespace ${NAME_SPACE}`, but not `helm uninstall ${RELEASE_NAME} -n ${NAME_SPACE}`. This approach keeps the namespace but uninstalls the RHDH.